packages/typespec-vscode/src/utils.ts

```diff
@@ -1,4 +1,26 @@
+import { Executable } from "vscode-languageclient/node.js";
+
 /** normalize / and \\ to / */
 export function normalizeSlash(str: string): string {
   return str.replaceAll(/\\/g, "/");
 }
+
+/**
+ *
+ * @param exe
+ * @param win32Only only use Shell when the process.platform is "win32"
```

Comment:
Where is this win32Only parameter used when calling this function?

Reply:
the param has a default value as "true", so all the callers just not provide the value for the 2nd parameter. Having this parameter here to make the util method more generic instead of something like "useShellInExecForWindows(...)". thanks.

Reply:
i get the purpose, though I also don't really see when you'd actually want to do this? 
The main issue i am pointing out is that this option is not actually used anywhere yet

Reply:
Do you mean the "shell" option won't be used in platform other than "windows"? I am not familiar with linux/mac. so if that's the case, then yes, the param is not necessary. Otherwise, I would prefer to keep the util method complete by itself instead of considering how it's used. i.e. I dont need to think about how it's implemented (only works when the platform is windows) when calling it to enable shell now or in the future. thanks.

Reply:
yeah was mostly I am not sure I was seeing the point of planning to add this positional flag for something we don't need yet.